src/vision.cs

```diff
@@ -202,6 +202,12 @@ enum VNDetectBarcodesRequestRevision : ulong {
 		[TV (15, 0), Mac (12, 0), iOS (15, 0)]
 		[MacCatalyst (15, 0)]
 		Two = 2,
+		[TV (16, 0), Mac (13, 0), iOS (16, 0)]
+		[MacCatalyst (16, 0)]
```

Comment:
This is _really_ minor, but there's no need for two lines here:

```suggestion
		[TV (16, 0), Mac (13, 0), iOS (16, 0), MacCatalyst (16, 0)]
```

Reply:
@rolfbjarne I was just matching the format of what was already here, but will modify the ones I added